tests/legacy-ray_charles.e2e.spec.js

```diff
@@ -302,7 +308,7 @@ function runTests (driver, modes) {
                     assert.equal('Ray Charles', title);
                     // If we have reached the last mode, quit the driver
                     if (mode === modes[modes.length - 1]) {
-                        await driver.quit();
+                        return driver.quit();
```

Comment:
Please check if there are any other `driver.quit();` in this file, and if there are, add a `return` before.

Reply:
done